### DISCUSSION TITLE
Home assistant doorbell events since 0.12

### DISCUSSION THREAD

**Discussion by ve6rah** [blakeblackshear/frigate#6130] [Ask A Question] [answered]
I have a really odd one here, and need some advice.

Ever since updating to 0.12 my dahua doorbell cameras are no longer sending button push events to Home Assistant, now at first glance this shouldn't have anything to do with frigate, and yet, whenever I shut down frigate everything works as before, but as soon as I launch frigate the doorbell stops sending the button press events. Other events from the cameras still come through to the HA event bus, but the "AlarmLocal" events no longer come through, and even the outdoor chime on button press doesn't work. the camera part of the device appears to be working normally, and frigate is having no issues.

Anyone have any hypothesis why the cameras would stop allowing the doorbell button to function when frigate is running? And only since the upgrade to 0.12 and transition to Go2RTC? And better yet, ideas on how to solve it?

**Comment by NickM-27** [comment, marked as answer]
this is covered in the docs and on go2rtc, it is because by default go2rtc enables backchannel audio (2 way talk) and this disables the button pressing.

Simply add `#backchannel=0` to the end of your doorbell rtsp stream in go2rtc config and it will fix it

**Comment by ve6rah** [comment]
And here I thought I had read the frigate docs thoroughly in this transition... apparently not!

That fixed it

Thanks for the super speedy reply!

I'm really enjoying some of the 0.12 improvements, I especially like how quickly gotortc gets that stream cast to my Google home display! before 0.12 it took so long that there was no point at all, now by the  time I look over at the display the video is already up!

Next step is to play with the snapshots and zones to get a better look at who's at the door for phone push notifications....
